### PR TITLE
[Update] refactor login

### PR DESCRIPTION
**Pull request by olololoe110399**
## Related Tickets

- [#Ticket ID](https://???)

## What's this PR do ?

- [ ] Implement function A
- [ ] Fix bugs on function B

## Library
*(List library, third party add new, include version)*

- Library: GoogleAnalysis v1.2.0

## Impacted Areas in Application
*(List features, api, models or services that this PR will affect)*

- foo/bar
- bar/baz

## Checklist

- [ ] Unit test run successfully?
- [ ] Updated library at [here](https://???)
- [ ] Fill link PR into ticket and the opposite
- [ ] Note reason, scope of influence, solution into ticket
- [ ] Validate UI/Model/API
- [ ] Checked on Chrome, Safari,..
- [ ] Followed latest specs at [#Link specs](https://???)

### CODE REVIEW

File: front-end/src/components/PrivateRoute.tsx [hunk 1]
```diff
@@ -3,20 +3,21 @@ import { Route, Redirect, RouteProps } from 'react-router-dom';
...
 type Props = RouteProps & {
   component: React.ComponentType<any>,
-  auth?: AuthState;
+  isAuth?: boolean;
 }
 
-const PrivateRoute = ({ component: Component, auth, ...rest }:Props) => (
-  <Route
+const PrivateRoute = ({ component: Component, isAuth, ...rest }: Props) => {
+  console.log("PrivateRoute", isAuth);
```
Comment (tranquocviet226) [inline front-end/src/components/PrivateRoute.tsx:13]: xoa console nay

File: front-end/src/components/UnAuthRoute.tsx [hunk 2]
```diff
@@ -10,21 +10,21 @@ import { connect } from "react-redux";
...
   path: RouteProps["path"];
   exact: RouteProps["exact"];
   component: React.ComponentType<any>;
-  auth?: AuthState;
+  isAuth: boolean;
 };
 
-const UnAuthRoute = ({ component: Component, auth, ...rest }: Props) => {
+const UnAuthRoute = ({ component: Component, isAuth, ...rest }: Props) => {
+  console.log("UnAuthRoute", isAuth);
```
Comment (tranquocviet226) [inline front-end/src/components/UnAuthRoute.tsx:22]: console nua nay

File: front-end/src/components/PrivateRoute.tsx (hunk 1 above)
Comment (olololoe110399) [reply front-end/src/components/PrivateRoute.tsx:13]: ok.

File: front-end/src/components/UnAuthRoute.tsx (hunk 2 above)
Comment (olololoe110399) [reply front-end/src/components/UnAuthRoute.tsx:22]: ok

**Pull request merged**